src/_pytest/reports.py

```diff
@@ -196,11 +196,17 @@ def head_line(self) -> Optional[str]:
             return domain
         return None
 
-    def _get_verbose_word(self, config: Config):
+    def _get_verbose_word(
```

Comment:
As the method does not return a single word anymore, it should probably be renamed to reflect what it does, really. Perhaps, smth like this:
```suggestion
    def _get_verbose_word_with_markup(
```

Reply:
Done.